core/themes/seven/css/style.css

```diff
@@ -694,6 +694,9 @@ fieldset .fieldset-wrapper {
   border-radius: 4px;
   background: transparent;
 }
+.js fieldset.collapsible .fieldset-legend {
+  font-size: 16px;
```

Comment:
All other font sizes in Seven are either `em` or `%`, we should be consistent here.

Reply:
fixed.